hack/e2e/run-rp-and-e2e.sh

```diff
@@ -59,14 +69,19 @@ register_sub() {
       "https://localhost:8443/subscriptions/$AZURE_SUBSCRIPTION_ID?api-version=2.0"
 }
 
-clean_e2e_db(){
+clean_e2e_db() {
     echo "########## 🧹 Deleting DB $DATABASE_NAME ##########"
     az cosmosdb sql database delete --name $DATABASE_NAME \
         --yes \
         --account-name $DATABASE_ACCOUNT_NAME \
         --resource-group $RESOURCEGROUP >/dev/null
 }
 
+delete_e2e_cluster() {
+    echo "########## 🧹 Deleting Cluster $CLUSTER ##########"
+    CLUSTER="v4-e2e-V$BUILD_BUILDID-$LOCATION" go run ./hack/cluster delete
```

Comment:
Setting the cluster to a pipeline-specific name before deleting it will break this function for local testing where `CLUSTER` can be anything. The script has logic to fail when sourcing it if `CLUSTER` is not set (lines 96-99 of this update) and will set `CLUSTER` to something similar to this if the `LOCAL_E2E` variable is not set (91-94), so the name should just be used from the existing environment instead of being re-assigned inside the function.

Reply:
Agreed, I'll change it.